contrib/completions/fish/makoctl.fish

```diff
@@ -0,0 +1,23 @@
+function __fish_makoctl_complete_no_subcommand
+	for i in (commandline -opc)
```

Comment:
Indentation looks a little weird here?

Reply:
I didn't notice. Changed all indentations to use tab now.